bundles/org.openhab.binding.mikrotik/pom.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <dependencies>
+    <dependency>
+      <groupId>me.legrange</groupId>
+      <artifactId>mikrotik</artifactId>
+      <version>3.0.7</version>
+    </dependency>
+  </dependencies>
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>3.1.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>org.openhab.binding.mikrotik</artifactId>
+  <name>openHAB Add-ons :: Bundles :: Mikrotik Binding</name>
+</project>
```

Comment:
Please format and order the tags like in the other bindings.

Reply:
Will use [bluetooth.generic's POM](/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.bluetooth.generic/pom.xml) as a sample.